### PR TITLE
turn off frozen object

### PR DESCRIPTION
**Pull request by hchoura**

### CODE REVIEW

**Review by nanoscott — APPROVED** [review]

File: src/foam/nanos/analytics/DAOFoldManager.js [hunk 1]
```diff
@@ -85,8 +85,6 @@ setCloseTimeExpr(new foam.glang.EndOfTimeSpan.Builder(getX())
             c.setCloseTime(id.getCloseTime());
             c.setKey(key);
             c.init_();
-          } else {
-            c = (Candlestick) c.fclone();
```
Comment (kgrgreer) [inline src/foam/nanos/analytics/DAOFoldManager.js:89]: If you aren't going to clone the object, then I don't think you need to put it back in the DAO either. Just move the getDao().put(c) line to the last line of the 'if' block above (line 87.5).

File: src/foam/nanos/analytics/Candlestick.js [hunk 2]
```diff
@@ -100,6 +100,16 @@ foam.CLASS({
     }
   ],
   methods: [
+    {
```
Comment (kgrgreer) [inline src/foam/nanos/analytics/Candlestick.js:103]: Are these changes needed if you're not cloneing or put()-ing below?

**Comment by jlhughes** [comment]
The initial put is needed and that will freeze the object, so believe we need the freeze/clone overrides. 
we most likely don't need the clone as we are no longer explicitly calling it.  But these objects are persisted and replayed.  (1 Hour and daily  OMs, PM, ...).

File: src/foam/nanos/analytics/Candlestick.js (hunk 2 above)
Comment (hchoura) [reply src/foam/nanos/analytics/Candlestick.js:103]: Yes.
